### PR TITLE
Display imported address somewhere

### PR DESCRIPTION
**Pull request by marknguyen85**

### CODE REVIEW

File: src/utils/helpers.js [hunk 1]
```diff
@@ -39,3 +39,27 @@ export function calculateMinConversionRate(slippageRate, expectedRate) {
 export function getDefaultAddress() {
   return "0x" + Array(41).join("0");
 }
+
+export function reverseStr(str) {
+  if (!str || str.length == 0) {
```
Comment (manhlx3006) [inline src/utils/helpers.js]: should use `===`

File: src/utils/helpers.js [hunk 2]
```diff
@@ -39,3 +39,27 @@ export function calculateMinConversionRate(slippageRate, expectedRate) {
...
+}
+
+export function formatAddress(fullAddress, length = 20) {
+  if (!fullAddress || fullAddress.length <= length) {
+    return fullAddress;
+  }
+
+  const lLength = 13;
+  const dotLength = 3;
+  const rLength = length - lLength - dotLength;
```
Comment (manhlx3006) [inline src/utils/helpers.js]: Is it possible that `length - lLength - dotLength <= 0`?

File: src/components/commons/InputGroup.js [hunk 3]
```diff
@@ -77,9 +77,14 @@ export default class InputGroup extends Component {
...
         <div className={"input-group__info"}>
           {this.props.isAccountImported && (
-            <div className={"common__flexbox"}>
-              Balance: {this.props.isBalanceLoading ? <div className={"input-group__loading common__loading"}/> : formatAmount(this.props.sourceToken.balance)} {this.props.sourceToken.symbol}
-            </div>
+            <Fragment>
+              <div className={"common__flexbox"}>
+                {this.props.isBalanceLoading ? <div className={"input-group__loading common__loading"}/> : formatAmount(this.props.sourceToken.balance)} {this.props.sourceToken.symbol}
+              </div>
+              <div className={"common__flexbox input-group__address"}>
```
Comment (manhlx3006) [inline src/components/commons/InputGroup.js:89]: We need to allow user click address and open `https://scan.testnet.tomochain.com/address/{address}` or `https://scan.testnet.tomochain.com/address/{address}` depends on environment

**Review by manhlx3006 — CHANGES_REQUESTED** [review]

File: src/utils/helpers.js (hunk 1 above)
Comment (marknguyen85) [reply src/utils/helpers.js]: ok

File: src/components/commons/InputGroup.js (hunk 3 above)
Comment (marknguyen85) [reply src/components/commons/InputGroup.js:89]: ok i forgot about this matter.

File: src/components/swap/SwapView.js [hunk 4]
```diff
@@ -42,7 +43,7 @@ export default class SwapView extends Component {
             </div>
 
             <div className={"input-group__info"}>
-              1 {this.props.sourceToken.symbol} = {isLoadingRateShown ? <div className={"input-group__loading common__loading"}/> : formatAmount(this.props.tokenPairRate)} {this.props.destToken.symbol}
+              {this.props.sourceToken.symbol} = {isLoadingRateShown ? <div className={"input-group__loading common__loading"}/> : formatAmount(this.props.tokenPairRate)} {this.props.destToken.symbol}
```
Comment (manhlx3006) [inline src/components/swap/SwapView.js]: why remove number `1`? 
Now it will be `TOMO = xxx TIIM` instead of `1 TOMO = xx TIIM`?

Comment (marknguyen85) [reply src/components/swap/SwapView.js]: ok i will revert it.

File: src/components/commons/InputGroup.js [hunk 5]
```diff
@@ -77,9 +82,14 @@ export default class InputGroup extends Component {
...
```
Comment (manhlx3006) [inline src/components/commons/InputGroup.js:89]: Please change color of address to the correct one as designed: color hex: `#aab1d4`

**Review by manhlx3006 — CHANGES_REQUESTED** [review]

File: src/components/commons/InputGroup.js (hunk 5 above)
Comment (marknguyen85) [reply src/components/commons/InputGroup.js:89]: changed

**Review by manhlx3006 — APPROVED** [review]

**Pull request merged**